views/pages/fixture_editor.js

```diff
@@ -30,81 +30,69 @@ module.exports = function(options) {
 
   // Existing manufacturer
   str += `<section v-if="fixture.useExistingManufacturer">`;
-  str += `<label class="validate-group">`;
-  str += `<span class="label">Choose from list</span>`;
-  str += `<span class="value">`;
-  str += `<select required v-model="fixture.manufacturerShortName" :class="{ empty: fixture.manufacturerShortName === '' }" ref="existingManufacturerSelect">`;
-  str += `<option value="">Please select a manufacturer</option>`;
-  for (const man of Object.keys(options.register.manufacturers)) {
-    str += `<option value="${man}">${options.manufacturers[man].name}</option>`;
-  }
-  str += `</select>`;
-  str += `<span class="error-message" hidden></span>`;
-  str += `</span>`;
-  str += `</label>`;
+  str += simpleLabel(
+    `Choose from list`,
+    [
+      `<select required v-model="fixture.manufacturerShortName" :class="{ empty: fixture.manufacturerShortName === '' }" ref="existingManufacturerSelect">`,
+      `<option value="">Please select a manufacturer</option>`,
+      Object.keys(options.register.manufacturers).map(
+        man => `<option value="${man}">${options.manufacturers[man].name}</option>`
+      ).join(``),
+      `</select>`
+    ].join(``)
+  );
   str += `<div class="button-bar">or <a href="#add-new-manufacturer" @click.prevent="switchManufacturer(false)">add a new manufacturer</a></div>`;
   str += `</section>`; // [v-if="fixture.useExistingManufacturer"]
 
   // New manufacturer
   str += `<div v-else>`;
   str += `<section class="new-manufacturer-name">`;
-  str += `<label class="validate-group">`;
-  str += `<span class="label">Name</span>`;
-  str += `<span class="value">`;
-  str += textInput(`fixture.newManufacturerName`, {
-    property: properties.manufacturer.name,
-    required: true,
-    attributes: { ref: `newManufacturerNameInput` }
-  });
-  str += `<span class="error-message" hidden></span>`;
-  str += `</span>`;
-  str += `</label>`;
+  str += simpleLabel(
```

Comment:
I'd prefer having `` `Name` `` and ``textInput(`fixture.newManufacturerName`, {`` in the same line. This way, we can save a lot of whitespace that does not increase readability.

This applies to all similar function calls. If the second argument is too long, better construct it in a local variable first.